src/App.tsx

```diff
@@ -37,10 +39,17 @@ const App = () => {
         <BrowserRouter>
           <Navbar></Navbar>
           <Routes>
```

Comment:
Podemos hacer esto para que el / al estar logeado sea directamente la ventana de restaurants.

```suggestion
          <Routes>
					  <Route path="/" element={<Navigate to="/restaurants" />}></Route>
```